lib/Conversation/styles.scss

```diff
@@ -0,0 +1,166 @@
+   Config
+   ========================================================================== */
+$conversation-border-radius: 5px !default;
+$conversation-padding: $layout-spacing-base !default;
+
+$conversation-border-width: 1px !default;
+$conversation-border-color: $color-input-border !default;
+
+$conversation-comment-body-padding: 0 0 0 ($person-avatar-size + $layout-spacing-base/2) !default;
+
+$conversation-meta-bullet-size: 4px !default;
+$conversation-meta-text-size: $typo-size-small !default;
+$conversation-meta-text-color: $color-brand-dark !default;
+
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.conversation {
+  border-radius: $conversation-border-radius;
+  border: $conversation-border-width solid transparent;
+
+  &:hover {
+    border-color: $conversation-border-color;
+  }
+}
+
+.conversation__foot {
+  display: none;
+  padding: $conversation-padding;
+}
+
+.conversation__resolve {
+  display: none;
+  text-align: right;
+  font-size: $typo-size-slight;
+
+  svg {
+    position: relative;
+    top: 2px;
+    margin: 0 $conversation-padding/4 0 0;
+  }
+}
+
+.conversation__comment {
+  position: relative;
+  display: none;
+  padding: $conversation-padding $conversation-padding 0;
+
+  &:first-child {
+    display: block;
+  }
+}
+
+.conversation__reply-count {
+  padding: 0 0 $conversation-padding $conversation-padding+$avatar-size;
+  font-size: $conversation-meta-text-size;
+}
+
+.conversation__comment-body {
+  margin: -$typo-size-base/2 0 0;
+  padding: $conversation-comment-body-padding;
+}
+
+.conversation__text {
+  margin: 0 0 $conversation-padding/4;
+}
+
+.conversation__meta-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.conversation__meta {
+  font-size: $conversation-meta-text-size;
+  margin: 0 $conversation-padding/2 0 0;
+
+  &:after {
+    content: "";
+    display: inline-block;
+    width: $conversation-meta-bullet-size;
+    height: $conversation-meta-bullet-size;
+    border-radius: $conversation-meta-bullet-size;
+    background: $conversation-meta-text-color;
+    margin: 0 0 0 $conversation-padding/2;
+    position: relative;
+    top: -2px;
+  }
+
+  &:last-child {
+    &:after {
+      display: none;
+    }
+  }
+}
+
+.conversation__focus-fallback {
+  position: absolute;
+  left: -999em;
+}
+
+.conversation__date-text {
+  color: $conversation-meta-text-color;
+}
+
+.conversation--is-active {
+  border-color: $conversation-border-color;
+
+  .conversation__resolve {
+    border-bottom: $conversation-border-width solid $conversation-border-color;
+  }
+
+  .conversation__comment {
+    display: block;
+    padding-bottom: $conversation-padding;
+    border-bottom: $conversation-border-width solid $conversation-border-color;
+  }
+
+  .conversation__foot,
+  .conversation__resolve,
+  .comment {
+    display: block;
+  }
+}
+
+
+// Form
+.form__input {
```

Comment:
Should we stick this in a form component scss file or maybe just prefix this with conversation?